pack.yaml

```diff
@@ -5,8 +5,9 @@ description: Zabbix Monitoring System
 keywords:
   - zabbix
   - monitoring
-version: 1.0.0
+version: 0.4.0
```

Comment:
Should I bump this pack version to 0.4.0 or is that not necessary since they already moved past it?

Reply:
lets do 0.4.1